dashboard/app/models/pd/application/facilitator1819_application.rb

```diff
@@ -50,12 +50,32 @@ def year
       YEAR_18_19
     end
 
+    ONLY_WEEKEND = "I will only be able to attend Saturday and Sunday of the training"
+
     # Are we still accepting applications?
     APPLICATION_CLOSE_DATE = Date.new(2018, 2, 1)
     def self.open?
       Time.zone.now < APPLICATION_CLOSE_DATE
     end
 
+    # @override
+    def self.options
+      super.merge(
+        {
+          csd_csp_teachercon_availability: [
+            'TeacherCon 1: June 17 - 22, 2018',
+            'TeacherCon 2: July 22 - 27, 2018',
+            TEXT_FIELDS[:not_available_for_teachercon]
+          ],
+          csd_csp_fit_availability: [
+            'June 23 - 24, 2018 (immediately following TeacherCon 1)',
+            'July 28 - 29, 2018 (immediately following TeacherCon 2)',
```

Comment:
Seems like these strings should be in the constants file

Reply:
These aren't shared constants - keeping them here is fine